core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -176,8 +176,12 @@ class BasicWorkflow extends EnsimeSpec
               BasicTypeInfo("Blue$", DeclaredAs.Object, "org.example.Blue$", List(), List(), Some(_)),
               BasicTypeInfo("CaseClassWithCamelCaseName", DeclaredAs.Class, "org.example.CaseClassWithCamelCaseName", List(), List(), Some(_)),
               BasicTypeInfo("CaseClassWithCamelCaseName$", DeclaredAs.Object, "org.example.CaseClassWithCamelCaseName$", List(), List(), Some(_)),
+              BasicTypeInfo("File", DeclaredAs.Class, "org.example.File", List(), List(), Some(_)),
```

Comment:
if you put your test file in a different package, you won't need to change this test. I recommend doing that, e.g. `org/user/mythings.scala`